<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RawTxWindow</class>
 <widget class="QWidget" name="RawTxWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>151</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>371</width>
    <height>151</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Raw Transmit</string>
  </property>
  <widget class="QPushButton" name="singleSendButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>87</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Send Single</string>
   </property>
  </widget>
  <widget class="QPushButton" name="repeatSendButton">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>95</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Send Repeat</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>31</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>ms</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fieldAddress">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>72</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>hhhhhhhh</string>
   </property>
   <property name="text">
    <string>00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>55</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Address</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="fieldDLC">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>8</number>
   </property>
   <property name="value">
    <number>8</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>28</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>DLC</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fieldByte0">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>hh</string>
   </property>
   <property name="text">
    <string>00</string>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>-1</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Payload</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="fieldByte1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>hh</string>
   </property>
   <property name="text">
    <string>00</string>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="fieldByte2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>hh</string>
   </property>
   <property name="text">
    <string>00</string>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="fieldByte3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>hh</string>
   </property>
   <property name="text">
    <string>00</string>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="fieldByte4">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>hh</string>
   </property>
   <property name="text">
    <string>00</string>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="fieldByte5">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>30</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>hh</string>
   </property>
   <property name="text">
    <string>00</string>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="fieldByte6">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string>hh</string>
   </property>
   <property name="text">
    <string>00</string>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLineEdit" name="fieldByte7">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>21</width>
     <height>20</height>
    </rect>
   </property>
   <property name="inputMask">
    <string extracomment="Hex Only">hh</string>
   </property>
   <property name="text">
    <string>00</string>
   </property>
   <property name="maxLength">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>50</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>50</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_RepeatRate">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>74</width>
     <height>26</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="value">
    <number>1000</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_IsRTR">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>54</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>RTR</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_IsExtended">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>109</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Extended ID</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_IsErrorFrame">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>106</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Error Frame</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
